app/models/manageiq/providers/azure/cloud_manager/metrics_capture.rb

```diff
@@ -34,8 +34,8 @@ class ManageIQ::Providers::Azure::CloudManager::MetricsCapture < ManageIQ::Provi
       :counter_key => 'disk_usage_rate_average', # TODO: should be 'disk_usage_absolute_average',
       :source      => :api,
       :metrics     => [
-        'Per Disk Read Bytes/sec',
-        'Per Disk Write Bytes/sec',
+        'Disk Read Bytes',
+        'Disk Write Bytes',
```

Comment:
@djberg96 since this is a rate (`disk_usage_rate_average`) I don't think that going to an absolute (Bytes) makes sense here.  How about `Disk Read Bytes/Sec` instead? (ref https://docs.microsoft.com/en-us/azure/azure-monitor/platform/metrics-supported#microsoftclassiccomputevirtualmachines)

Reply:
@agrare That's classic compute (i.e. the "old" azure). The current metrics list is:
```
- "Percentage CPU",
- "Network In",
- "Network Out",
- "Disk Read Bytes",
- "Disk Write Bytes",
- "Disk Read Operations/Sec",
- "Disk Write Operations/Sec",
- "CPU Credits Remaining",
- "CPU Credits Consumed",
- "Per Disk Read Bytes/sec",
- "Per Disk Write Bytes/sec",
- "Per Disk Read Operations/Sec",
- "Per Disk Write Operations/Sec",
- "Per Disk QD",
- "OS Per Disk Read Bytes/sec",
- "OS Per Disk Write Bytes/sec",
- "OS Per Disk Read Operations/Sec",
- "OS Per Disk Write Operations/Sec",
- "OS Per Disk QD",
- "Data Disk Read Bytes/sec",
- "Data Disk Write Bytes/sec",
-  "Data Disk Read Operations/Sec",
- "Data Disk Write Operations/Sec",
- "Data Disk Queue Depth",
- "OS Disk Read Bytes/sec",
- "OS Disk Write Bytes/sec",
- "OS Disk Read Operations/Sec",
- "OS Disk Write Operations/Sec",
- "OS Disk Queue Depth",
- "Inbound Flows",
- "Outbound Flows",
- "Inbound Flows Maximum Creation Rate",
- "Outbound Flows Maximum Creation Rate",
- "Premium Data Disk Cache Read Hit",
- "Premium Data Disk Cache Read Miss",
- "Premium OS Disk Cache Read Hit",
- "Premium OS Disk Cache Read Miss",
- "Network In Total",
- "Network Out Total"
```

Reply:
https://docs.microsoft.com/en-us/azure/azure-monitor/platform/metrics-supported#microsoftcomputevirtualmachines

Reply:
Okay we still can't stick an absolute value in a rate column though, it looks like they split it up into "(OS Disk|Data Disk) Read Bytes/sec"?

Reply:
@agrare Ok, updated, though now I'm getting a spec failure that I'm having trouble nailing down. It seems to not use the staged data if there's more than 2 metrics.

Reply:
@djberg96 I think the values returned are just doubled whatever the spec is expecting since it is just summing all the stubbed counters